### ISSUE TITLE
Accessibility: Focus order doesn’t follow expectations

### ISSUE THREAD

**Issue by aparkening** [alistapart/AListApart#214]
From Andrew's accessibility review:

Can the top nav be at the top of the DOM?  The result for end users is that they tab directly into the main content, and then the top nav isn’t focused until closer to the end of the page.  The behavior is actually different on different pages:

1. Home page, events, blog, columns: 
- Main content
- Top nav
- Footer (about, authors, etc)

2. Article pages, write for us page:
- Main content
- Comments
- Footer
- Top nav

3. The keyboard navigation should be:
- Top nav
- Article content, in the order presented
- Footer

The navigation is nearly correct now except that the top nav is always out of place. 

This is a WCAG 2.0 3.2.3 Failure (Consistent navigation)

**Comment by murtaugh** [comment]
@michellekondou I re-ordered the source last week — do you think we should take the big "A LIST APART" out of the tab order altogether? The laurel link is the exact same thing.

**Comment by murtaugh** [comment]
I reordered the source, and the nav is now the first thing that's tabbed through.
